#4411 docs: make build steps for local development more easily discoverable

knittl opened the pull request:
Provide build instructions directly in the `CONTRIBUTING.md` file of the repository and make the link to build documentation stand out more.

Closes #4411 
Closes #3929

deviantony commented on CONTRIBUTING.md:
```diff
@@ -74,3 +74,23 @@ Our contribution process is described below. Some of the steps can be visualized
+
+Portainer can now be accessed at <http://localhost:9000>.
+
+Find more detailed steps at <https://www.portainer.io/documentation/how-to-contribute/>.
```
```suggestion
Find more detailed steps at <https://documentation.portainer.io/contributing/instructions/>.
```

deviantony commented on README.md:
```diff
@@ -30,6 +30,7 @@ Unlike the public demo, the playground sessions are deleted after 4 hours. Apart
 
 - [Deploy Portainer](https://www.portainer.io/installation/)
 - [Documentation](https://documentation.portainer.io)
+- [Building Portainer](https://www.portainer.io/documentation/how-to-contribute/)
```
```suggestion
- [Building Portainer](https://documentation.portainer.io/contributing/instructions/)
```

deviantony reviewed (changes requested):
@knittl thanks for waiting, we've finally updated our contribution documentation, I've suggested an edit to the links in your changes.

knittl commented:
> @knittl thanks for waiting, we've finally updated our contribution documentation, I've suggested an edit to the links in your changes.

Thanks, I've incorporated your suggestions and updated the links!

deviantony reviewed (approved):
LGTM

knittl commented:
Ping. What's the status of this? Is there anything missing from my side?

deviantony commented:
Sorry @knittl, it got lost in our contribution merging process. I'll take care of it.

deviantony commented:
Thanks @knittl :+1:
